docs/user/building-images.md

```diff
@@ -97,6 +97,11 @@ RUN dnf -y copr enable @redhat-et/flightctl centos-stream-9-x86_64 && \
     dnf -y clean all && \
     systemctl enable flightctl-agent.service
 
+# compose application support
```

Comment:
I’d like to ensure users users can distinguish between minimum requirement and options, please.

Could we clarify in the comments something lime “# Optionally, if you want support for podman-compse applications, install podman-compose / uncomment below”?

Reply:
Also, should we then copy to getting started.md, too?

Reply:
updated